packages/amplication-client/src/Workspaces/hooks/useResources.ts

```diff
@@ -34,7 +34,8 @@ const createGitRepositoryFullName = (
 
 const useResources = (
   currentWorkspace: models.Workspace | undefined,
-  currentProject: models.Project | undefined
+  currentProject: models.Project | undefined,
+  addBlock: (id: string) => void
```

Comment:
we need to be consistent with the way we already use addEntity (see below `addEntity`)